make/compiler/Makefile.cray-prgenv-cray

```diff
@@ -15,8 +15,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Suppress warning about variables used before defined
-CRAYPE_GEN_CFLAGS = -hnomessage=7212
+# Suppress warnings:
+# 7212: Suppress warning about variables used before defined
+# 167:  Suppress warning about pointer mismatches (required by 'c_fn_ptr')
```

Comment:
Not that it really matters, but I'd prefer to have the warning numbers listed in increasing order.